modules/single-node-asg/variables.tf

```diff
@@ -115,3 +115,40 @@ variable "load_balancers" {
   description = "The list of load balancers names to pass to the ASG module"
   type        = "list"
 }
+
+## DLM variables
+variable "dlm_create_dlm_iam_role" {
+  description = "Creates DLM IAm role and permissions to create snapshots, the role can be create one time pero AWS account"
+  default     = "false"
+}
+
+variable "dml_description" {
+  description = "DLM lifecycle policy description"
+  default     = "DLM lifecycle policy"
+}
+
+variable "dlm_ebs_target_tags" {
+  description = "Tags to filter the volume that we want to take the snapshot."
+  default     = {}
```

Comment:
Is this a required parameter? Or is this an additional filter? Eg, will the module work with this default, and take snapshots, or is this parameter required for the module to take snapshots?